lib/src/screens/persistence_store/app_persistence_store_screen.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
```

Comment:
Thinking if we need separate screen for Persistence. The only difference between cache/persistence is the box name. I think we can provide 1 more input Cache/Persistence and reuse the same screen.